### ISSUE TITLE
Detecting module when using AWS Codecommit repository as source

### ISSUE THREAD

**Issue by ghost** [VladRassokhin/intellij-hcl#163]
### Installation details
 - [ ] IntelliJ IDEA 2018.1.5 (Community Edition)
Build #IC-181.5281.24, built on June 12, 2018
JRE: 1.8.0_152-release-1136-b39 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
 - [ ] intellij-hcl plugin version 0.6.12
 - [ ] Terraform version (v0.11.7`)

### Terraform Configuration Files
```hcl
module "lambda_app" {
  source                      = "git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/mytestrepo//src/modules/lambda?ref=v0.1.0"
  app_name                    = "xxxx"
  description                 = "xxxx"
  timeout                        = 10
}
```

### Expected Behavior
The IDE should have detected the module from codecommit after running terraform get command

### Actual Behavior
The IDE does not detected the module nor the module outputs 

### Steps to Reproduce
1. Create a aws codecommit repo  and added some terraform modules
2. Use the module from the codecommit repo in a terrraform file as show n above

**Comment by VladRassokhin** [comment]
Could you please add content of `.terraform/modules/modules.json` here? I'm interested in part which contains your source id.

**Comment by brajivbala** [comment]
@VladRassokhin  the module.json file looks as below
```json
{
   "Modules":[
      {
         "Source":"git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/<reponame>//src/modules/cloudwatch_kinesis_monitoring?ref=v0.0.8",
         "Key":"1.kinesis_stream_monitoring;git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/<reponame>//src/modules/cloudwatch_kinesis_monitoring?ref=v0.0.8",
         "Version":"",
         "Dir":".terraform\\modules\\9d259ec562255e7d506220b13856f71e",
         "Root":"src\\modules\\cloudwatch_kinesis_monitoring"
      },
      {
         "Source":"git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/<reponame>//src/modules/lambda?ref=v0.0.8",
         "Key":"1.lambda_app;git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/<reponame>//src/modules/lambda?ref=v0.0.8",
         "Version":"",
         "Dir":".terraform\\modules\\7fadb607477b1833466779c976041ce7",
         "Root":"src\\modules\\lambda"
      },
      {
         "Source":"../cloudwatch_lambda_monitoring",
         "Key":"1.lambda_app;git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/<reponame>//src/modules/lambda?ref=v0.0.8|lambda_app_cloudwatch;../cloudwatch_lambda_monitoring",
         "Version":"",
         "Dir":".terraform\\modules\\bdf827d588683123b90eafddb4a39fb2",
         "Root":""
      }
   ]
}
```

**Comment by VladRassokhin** [comment]
AFAIU there's no module with source `ref=v0.1.0` defined in modules.json but one with `ref=v0.0.8`. Please run `terraform init` and ensure there would be record in modules.json with required `source`

**Comment by brajivbala** [comment]
@VladRassokhin  I am indeed using the correct ref.. what i posted was a reference. I am able to download the module from codecommit properly. I ran deleted the local modules folder and rerean terraform init and terraform get commands.  The plugin doesn not recognize the module yet.

``` json
module "lambda_app" {
  source                      = "git::https://git-codecommit.eu-west-1.amazonaws.com/v1/repos/cig-tf-modules//src/modules/lambda?ref=v0.0.8"
  app_name                    = "lambda"
  description                 = "lambda to ingest data "
  project                     = "${var.project_name}"
  environment                 = "${var.environment}"
  iam_policy_document         = "${data.aws_iam_policy_document.lambda_iam_role.json}"
  assume_role_policy_document = "${data.aws_iam_policy_document.lambda_assume_role.json}"

  environment_variables = {
    RAW_BUCKET = "${aws_s3_bucket.raw_bucket.id}"
  }
```

**Comment by ExalDraen** [comment]
I believe this issue may be a more general issue with using `https` with `git` sources - I ran into the same issue with github:

When declaring a module with `git::https://github.com....` as `source` the IDE highlights the module and says the source is missing. However, modules.json shows that the module is in fact installed.

See below for full details.

### Terraform source
```
module "nomad_and_consul_servers" {
  source = "git::https://github.com/hashicorp/terraform-google-consul.git//modules/consul-cluster?ref=v0.2.0"

  gcp_region = "${var.gcp_region}"
  #...
}

```

### module.json
```
{
"Modules":[
    {"Source":"github.com/hashicorp/terraform-google-consul.git//modules/consul-cluster?ref=v0.2.0",
     "Key":"1.nomad_and_consul_servers;github.com/hashicorp/terraform-google-consul.git//modules/consul-cluster?ref=v0.2.0",
     "Version":"",
     "Dir":".terraform/modules/22c6f959945f3bfa514d8e831b21c42f",
     "Root":"modules/consul-cluster"},
    {"Source":"modules/nomad-firewall-rules",
     "Key":"1.nomad_firewall_rules;modules/nomad-firewall-rules",
     "Version":"",
     "Dir":".terraform/modules/1f0fc681ce873fd5e37d2017d38fc953",
     "Root":""},
    {"Source":"modules/nomad-cluster",
     "Key":"1.nomad_clients;modules/nomad-cluster",
     "Version":"",
     "Dir":".terraform/modules/14324e2a9ad1b1540b52f70fce2a0025",
     "Root":""},
    {"Source":"../nomad-firewall-rules",
     "Key":"1.nomad_clients;modules/nomad-cluster|firewall_rules;../nomad-firewall-rules",
     "Version":"",
     "Dir":".terraform/modules/6d18ccfe735393c8f3df478b094b0255",
     "Root":""},
    {"Source":"git::https://github.com/hashicorp/terraform-google-consul.git//modules/consul-cluster?ref=v0.2.0",
     "Key":"1.nomad_and_consul_servers;git::https://github.com/hashicorp/terraform-google-consul.git//modules/consul-cluster?ref=v0.2.0",
     "Version":"",
     "Dir":".terraform/modules/dbbd79baae0b3bd83d8824315430dc09",
     "Root":"modules/consul-cluster"}
    ]
}

```

### IDE info
```
PyCharm 2018.2.4 (Community Edition)
Build #PC-182.4505.26, built on September 19, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.17.14-202.fc28.x86_64
```

### Plugin
version `0.6.12`

**Comment by serialseb** [comment]
Same issue here

**Comment by VladRassokhin** [comment]
Most probably it's fixed in 0.6.14, please try and report

**Comment by VladRassokhin** [comment]
I've tested 0.6.14 with module like
```terraform
module "test-codecommit" {
  source = "git::ssh://git-codecommit.eu-west-1.amazonaws.com/v1/repos/test-terraform-module"
}
```
and it works fine.
